services/core-web/src/components/mine/Projects/ProjectSummary.tsx

```diff
@@ -215,19 +215,15 @@ export const ProjectSummary: FC = () => {
 
     if (!status_code || isNewProject) {
       status_code = "DFT";
-    } else if (!newActiveTab) {
-      if (isCore) {
-        status_code = formValues.status_code;
-      } else {
-        status_code = "SUB";
-      }
+    } else if (!newActiveTab && status_code === "DFT") {
+      status_code = "SUB";
```

Comment:
Noticed a sonarcloud issue- just some unused stuff that wants to be removed (isCore wants to be removed, and then probably related imports like the SystemFlagEnum and the selector)

Reply:
This should be removed now.